### PR TITLE
master

### PR DESCRIPTION
**Pull request by EngrTaofeek**
Increased the width of the button so as to display the language text in the button completely.

### CODE REVIEW

**Review by Elforama — CHANGES_REQUESTED** [review]
Thanks for the PR!

Can you please update the language button to use the DepthButton? It doesn't need to rely on TextButton anymore. All buttons will move towards depthbutton

File: build.gradle [hunk 1]
```diff
@@ -9,9 +9,9 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.0-alpha14'
+        classpath 'com.android.tools.build:gradle:7.0.0-alpha15'
         classpath 'com.google.gms:google-services:4.3.3'
-        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.4.1'
+        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.5.2'
```
Comment (Elforama) [inline build.gradle:14]: We don't need to change gradle. If this is required to build it locally, just don't check it into git.

**Comment by EngrTaofeek** [comment]
I forgot to uncheck the gradle file while I made the commit. I would switch the language button to DepthButtons and make a pull request

**Comment by Elforama** [comment]
Hey @EngrTaofeek, it looks like you need to rebase this branch to fix the merge conflicts.

**Pull request closed without merging**